gtsfm/two_view_estimator.py

```diff
@@ -111,15 +111,14 @@ def compute_relative_pose_metrics(
         i2Ti1_expected: expected relative pose.
 
     Returns:
-        Rotation error.
-        Unit translation error.
+        Rotation error, in degrees
+        Unit translation error, in degrees
     """
+    R_error_deg = comp_utils.compute_relative_rotation_angle(i2Ri1_computed, i2Ti1_expected.rotation())
 
-    R_error = comp_utils.compute_relative_rotation_angle(i2Ri1_computed, i2Ti1_expected.rotation())
+    U_error_deg = comp_utils.compute_relative_unit_translation_angle(i2Ui1_computed, Unit3(i2Ti1_expected.translation()))
 
-    U_error = comp_utils.compute_relative_unit_translation_angle(i2Ui1_computed, Unit3(i2Ti1_expected.translation()))
+    logger.debug(f"[Two View Estimator] Relative rotation error {R_error_deg:.2f} deg.")
```

Comment:
using `{}` for variables creates a lint error in logging. We should use the lazy formatting scheme: https://medium.com/swlh/why-it-matters-how-you-log-in-python-1a1085851205